scripts/mkcloud

```diff
@@ -675,6 +675,11 @@ function restartcloud()
 # bring up VMs that will take cloud controller/compute/storage roles
 function setupnodes()
 {
+    local nodenumbercontroller=1
+    if [[ $clusterconfig == *"services"* ]]; then
```

Comment:
I don't think this can be right.  The cluster names in `clusterconfig` are freeform text, so I could call my clusters `foo`, `bar`, and `qux` if I wanted.